test/hts-precompile/token-create/tokenCreateCustomContract.js

```diff
@@ -0,0 +1,391 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const utils = require('../utils')
+const { expect } = require('chai')
+const Constants = require('../../constants')
+const { expectValidHash } = require('../assertions')
+
+describe.only('TokenCreateCustomContract Test Suite', () => {
```

Comment:
What's the only here for?
I think this is breaking CI

Reply:
Ahh I forgot to remove it during the clean up. Thanks for the catch!